.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: podman-next ${{ github.sha }}
+  IMAGE_REGISTRY: quay.io/podman
+  REGISTRY_USER: ${{ secrets.QUAY_PODMAN_USERNAME }}
+  REGISTRY_PASSWORD: ${{ secrets.QUAY.PODMAN_PASSWORD }}
+  COPR_OWNER: rhcontainerbot
+  COPR_PROJECT: podman-next
+
+jobs:
+  wait-for-task:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
```

Comment:
Minor: Consider moving this to right before `Build FCOS Image`.  Only reason is it flows logically with those tasks.  It will also help with future refactoring if you want/need to turn the wait-for-copr part into a dedicated action (i.e. so it's not doing the pip install on every single run).

Reply:
changing in update.